test.disabled/ejabberd_tests/tests/rest_SUITE.erl

```diff
@@ -261,10 +264,113 @@ password_can_be_changed(Config) ->
     end),
     ok.
 
+list_contacts(Config) ->
+    escalus:fresh_story(
+        Config, [{alice, 1}, {bob, 1}],
+        fun(Alice, Bob) ->
+            AliceJID = escalus_utils:jid_to_lower(
+                            escalus_client:short_jid(Alice)),
+            BobJID = escalus_utils:jid_to_lower(escalus_client:short_jid(Bob)),
+            add_sample_contact(Bob, Alice),
+            % list bob's contacts
+            {?OK, R} = gett(lists:flatten(["/contacts/",
+                                           binary_to_list(BobJID)])),
+            [R1] =  decode_maplist(R),
+            #{jid := AliceJID, subscription := <<"none">>,
+                ask := <<"none">>} = R1,
+            ok
+        end
+    ),
+    ok.
+
+befriend_and_alienate(Config) ->
+    escalus:fresh_story(
+        Config, [{alice, 1}, {bob, 1}],
+        fun(Alice, Bob) ->
```

Comment:
Since both `Alice` and `Bob` are online over XMPP their clients should get some roster updates from the server. It'd be good to verify this.